shippable.resources.yml

```diff
@@ -63,22 +63,22 @@ resources:
   # config for GKE CLI
   - name: dmsk_cliConfig
     type: cliConfig
-    integration: drship_gke
+    integration: drship_gke     #replace with your Google Cloud integration name
```

Comment:
we need to change the integration name here to the new google cloud integration. Not sure what the name should be so havent' changed till yet